real_robot_dEE_impedance.py

```diff
@@ -31,20 +32,24 @@ def main():
     for episode in range(episodes):
         
         print("Episode: {}; Elapsed Time: {} minutes".format(episode, round((time.time()-start_time)/60), 4))
+        
         obs = env.reset()
+        
+        
         acc_reward = 0
         for step in range(steps):
             position = np.random.normal(loc=0.0, scale=0.1, size=(3,))
             stiffness_linear =  np.random.normal(loc=0.0, scale=10, size=(1,))
             stiffness_rot = np.random.normal(loc=0.0, scale=1, size=(1,))
             action = np.concatenate((position,stiffness_linear,stiffness_rot))
-            action = env.act_optimally()
-
-            obs, reward, done, info = env.step(action)  
+            # action = env.act_optimally()
+            # print(action)
 
-            if done:                
-                obs = env.reset()
-                continue
+            # obs, reward, done, info = env.step(action)
+            # obs = env.reset()
+            # if done:                
+            #     obs = env.reset()
+            #     continue
                 
             # if render:
```

Comment:
The environment doesn't step or reset anymore, I think he accidentally ruined this